solutions/fully-configurable/main.tf

```diff
@@ -32,3 +42,139 @@ module "scc_wp" {
   scc_wp_service_plan           = var.scc_workload_protection_service_plan
   cbr_rules                     = var.cbr_rules
 }
+
+########################################################################################################################
+# Beginning of Cloud Security Posture Management (CSPM) Configuration
+########################################################################################################################
+# SCC Workload Protection Trusted Profile
+########################################################################################################################
+
+# Create Trusted profile for SCC Workload Protection instance
+module "trusted_profile_scc_wp" {
+  count                       = var.cspm_enabled ? 1 : 0
+  source                      = "terraform-ibm-modules/trusted-profile/ibm"
+  version                     = "2.1.1"
+  trusted_profile_name        = local.scc_workload_protection_trusted_profile_name
+  trusted_profile_description = "Trusted Profile for SCC-WP to access App Config"
```

Comment:
Suggest go with:

"Trusted Profile for SCC workload Protection instance ${module.scc_wp.crn.guid} with required access for configuration aggregator"